From 336d4ab90e56cbb793c4f6899ff392e99deb5167 Mon Sep 17 00:00:00 2001
From: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
Date: Thu, 31 May 2018 21:18:27 +1000
Subject: [PATCH 07/65] docs: Tidying of ps,kill and skill manpages

Some minor tidying of these three man pages using more of the modern
(ish) macros that have been around for a while.

References:
 https://bugs.debian.org/893452
 https://bugs.debian.org/893457
 https://bugs.debian.org/894480

Signed-off-by: Craig Small <csmall@enc.com.au>
---
 kill.1  |  29 ++++---
 ps/ps.1 | 304 +++++++++++++++++++++++++++++++++-------------------------------
 skill.1 |  32 +++----
 3 files changed, 189 insertions(+), 176 deletions(-)

diff --git a/kill.1 b/kill.1
index a487f91..dd53040 100644
--- a/kill.1
+++ b/kill.1
@@ -1,11 +1,12 @@
-'\" t
-.\" (The preceding line is a note to broken versions of man to tell
-.\" them to pre-process this man page with tbl)
-.\" Man page for kill.
-.\" Licensed under version 2 of the GNU General Public License.
-.\" Written by Albert Cahalan; converted to a man page by
-.\" Michael K. Johnson
-.TH KILL 1 "October 2011" "procps-ng" "User Commands"
+.ig
+Written by Albert Cahalan, converted to a man page by Michael K. Johnson
+
+This manpage is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+..
+.TH KILL 1 "2018-05-31" "procps-ng" "User Commands"
 .SH NAME
 kill \- send a signal to a process
 .SH SYNOPSIS
@@ -64,7 +65,7 @@ Kill all processes you can kill.
 .B kill \-l 11
 Translate number 11 into a signal name.
 .TP
-.B kill -L
+.B kill \-L
 List the available signal choices in a nice table.
 .TP
 .B kill 123 543 2341 3453
@@ -78,16 +79,16 @@ Send the default signal, SIGTERM, to all those processes.
 .BR signal (7),
 .BR skill (1)
 .SH STANDARDS
-This command meets appropriate standards. The
+This command meets appropriate standards.  The
 .B \-L
 flag is Linux-specific.
 .SH AUTHOR
-.UR albert@users.sf.net
+.MT albert@users.sf.net
 Albert Cahalan
-.UE
+.ME
 wrote kill in 1999 to replace a bsdutils one that was not standards
 compliant.  The util-linux one might also work correctly.
 .SH "REPORTING BUGS"
 Please send bug reports to
-.UR procps@freelists.org
-.UE
+.MT procps@freelists.org
+.ME
diff --git a/ps/ps.1 b/ps/ps.1
index df3ded0..b358d2f 100644
--- a/ps/ps.1
+++ b/ps/ps.1
@@ -4,7 +4,7 @@
 .\" Quick hack conversion by Albert Cahalan, 1998.
 .\" Licensed under version 2 of the Gnu General Public License.
 .\"
-.TH PS 1 2018-01-13 "procps-ng" "User Commands"
+.TH PS 1 2018-05-31 "procps-ng" "User Commands"
 .\"
 .\" To render this page:
 .\"    groff -t -b -man -X -P-resolution -P100 -Tps ps.1 &
@@ -27,9 +27,7 @@
 .SH NAME
 ps \- report a snapshot of the current processes.
 .SH SYNOPSIS
-\fBps\fR [\fIoptions\fR]
-.PP
-.PP
+\fBps\fR [\,\fIoptions\/\fR]
 .SH DESCRIPTION
 .B ps
 displays information about a selection of the active processes.  If you want
@@ -81,7 +79,7 @@ The use of BSD\-style options will add process state (stat=STAT) to the
 default display and show the command args (args=COMMAND) instead of the
 executable name.  You can override this with the
 .B PS_FORMAT
-environment variable. The use of BSD\-style options will also change the
+environment variable.  The use of BSD\-style options will also change the
 process selection to include processes on other terminals (TTYs) that are
 owned by you; alternately, this may be described as setting the selection to
 be the set of all processes filtered to exclude processes owned by other
@@ -94,7 +92,6 @@ Except as described below, process selection options are additive.  The
 default selection is discarded, and then the selected processes are added to
 the set of processes to be displayed.  A process will thus be shown if it
 meets any of the given selection criteria.
-.PP
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 .SH "EXAMPLES"
 .TP 3
@@ -113,21 +110,21 @@ To see every process on the system using BSD syntax:
 .B ps\ axu
 .TP
 To print a process tree:
-.B ps\ -ejH
+.B ps\ \-ejH
 .br
 .B ps\ axjf
 .TP
 To get info about threads:
-.B ps\ -eLf
+.B ps\ \-eLf
 .br
 .B ps\ axms
 .TP
 To get security info:
-.B ps\ -eo euser,ruser,suser,fuser,f,comm,label
+.B ps\ \-eo euser,ruser,suser,fuser,f,comm,label
 .br
 .B ps\ axZ
 .br
-.B ps\ -eM
+.B ps\ \-eM
 .TP
 To see every process running as root (real\ &\ effective\ ID) in user format:
 .B ps\ \-U\ root\ \-u\ root\ u
@@ -144,12 +141,10 @@ Print only the process IDs of syslogd:
 .TP
 Print only the name of PID 42:
 .B ps\ \-q\ 42\ \-o\ comm=
-.PP
-.PP
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 .SH "SIMPLE PROCESS SELECTION"
 .TP
-.BR a
+.B a
 Lift the BSD\-style "only yourself" restriction, which is imposed upon the
 set of all processes when some BSD\-style (without "\-") options are used or
 when the
@@ -224,17 +219,18 @@ or to list all processes when used together with the
 option.
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 .PD
-.PP
 .SH "PROCESS SELECTION BY LIST"
 These options accept a single argument in the form of a blank\-separated or
 comma\-separated list.  They can be used multiple times.  For example:
-\fBps\ \-p\ "1\ 2"\ \-p\ 3,4\fR
+.B ps\ \-p\ "1\ 2"\ \-p\ 3,4
 .TP
 .RI \- 123
-Identical to \fB\-\-pid\ \fI123\fR.
+Identical to
+.BI \-\-pid\  123 \fR.
 .TP
 .I 123
-Identical to \fB\-\-pid\ \fI123\fR.
+Identical to
+.BI \-\-pid\  123 \fR.
 .TP
 .BI \-C \ cmdlist
 Select by command name.  This selects the processes whose executable name is
@@ -312,13 +308,16 @@ and
 .BR \-\-quick\-pid .
 .TP
 .BI \-q \ pidlist
-Select by PID (quick mode).  This selects the processes whose process ID numbers appear in
+Select by PID (quick mode).
+This selects the processes whose process ID numbers appear in
 .IR pidlist .
 With this option \fBps\fR reads the necessary info only
-for the pids listed in the \fIpidlist\fR and doesn't apply
-additional filtering rules. The order of pids is unsorted
-and preserved. No additional selection options, sorting
-and forest type listings are allowed in this mode.
+for the pids listed in the
+.I pidlist
+and doesn't apply additional filtering rules.
+The order of pids is unsorted and preserved.
+No additional selection options,
+sorting and forest type listings are allowed in this mode.
 Identical to
 .B q
 and
@@ -413,12 +412,10 @@ and
 .BR U .
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 .PD
-.PP
 .SH "OUTPUT FORMAT CONTROL"
 These options are used to choose the information displayed by
 .BR ps .
 The output may differ by personality.
-.PP
 .TP
 .B \-c
 Show different scheduler information for the
@@ -429,7 +426,7 @@ option.
 Display security context format (for SELinux).
 .TP
 .B \-f
-Do full\-format listing. This option can be combined with many other
+Do full\-format listing.  This option can be combined with many other
 UNIX\-style options to add additional columns.  It also causes the command
 arguments to be printed.  When used with
 .BR \-L ,
@@ -495,8 +492,9 @@ with the BSD personality.
 Like
 .BR \-o ,
 but preloaded with some default columns.  Identical to
-\fB\-o\ pid,\:\fIformat\fB,\:state,\:tname,\:time,\:command\fR or
-\fB\-o\ pid,\:\fIformat\fB,\:tname,\:time,\:cmd\fR,
+.BI \-o\ pid,\: format ,\:state,\:tname,\:time,\:command
+or
+.BI \-o\ pid,\: format ,\:tname,\:time,\:cmd \fR,
 see
 .B \-o
 below.
@@ -526,7 +524,7 @@ Explicit width
 control
 .RB ( "ps opid,\:wchan:42,\:cmd" )
 is offered too.  The behavior of
-.B ps -o pid=X,\:comm=Y
+.B ps \-o pid=X,\:comm=Y
 varies with personality; output may be one column named "X,\:comm=Y" or two
 columns named "X" and "Y".  Use multiple
 .B \-o
@@ -558,7 +556,6 @@ Add a column of security data.  Identical to
 (for SELinux).
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 .PD
-.PP
 .SH "OUTPUT MODIFIERS"
 .\"  .TP
 .\"  .B C
@@ -626,7 +623,10 @@ Repeat header lines, one per page of output.
 .TP
 .BI k \ spec
 Specify sorting order.  Sorting syntax is
-[\fB+\fR|\fB\-\fR]\fIkey\fR[,[\fB+\fR|\fB\-\fR]\fIkey\fR[,...]].
+.RB [ + | \- ]\c
+.I key\/\c
+.RB [,[ + | \- ]\c
+.IR key [,...]].
 Choose a multi\-letter key from the
 .B STANDARD FORMAT SPECIFIERS
 section.  The "+" is optional since default direction is increasing
@@ -671,10 +671,13 @@ or
 For sorting, obsolete BSD
 .B O
 option syntax is
-\fBO\fR[\fB+\fR|\fB\-\fR]\fIk1\fR[,[\fB+\fR|\fB\-\fR]\fIk2\fR[,...]].
+.BR O [ + | \- ]\c
+.IR k1 [,[\c
+.BR + | \- ]\c
+.IR k2 [,...]].
 It orders the processes listing according to the multilevel sort specified by
 the sequence of one\-letter short keys
-.IR k1 , k2 ", ..."
+.IR k1 , k2 ", ...\&"
 described in the
 .B OBSOLETE SORT KEYS
 section below.  The\ "+" is currently optional, merely re\-iterating the
@@ -694,8 +697,11 @@ repeatedly forks off short\-lived children to do work.
 .TP
 .BI \-\-sort \ spec
 Specify sorting order.  Sorting syntax is
-[\fB+\fR|\fB\-\fR]\fIkey\fR[,[\fB+\fR|\fB\-\fR]\fIkey\fR[,...]].  Choose a
-multi\-letter key from the
+.RI [ + | \- ]\c
+.IR key [,[\c
+.BR + | \- ]\c
+.IR key [,...]].
+Choose a multi\-letter key from the
 .B STANDARD FORMAT SPECIFIERS
 section.  The "+" is optional since default direction is increasing numerical
 or lexicographic order.  Identical to
@@ -713,7 +719,6 @@ Wide output.  Use this option twice for unlimited width.
 Set screen width.
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 .PD
-.PP
 .SH "THREAD DISPLAY"
 .TP
 .B H
@@ -731,19 +736,18 @@ Show threads after processes.
 .B \-T
 Show threads, possibly with SPID column.
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-.PD
-.PP
 .SH "OTHER INFORMATION"
 .TP
 .BI \-\-help \ section
 Print a help message.  The section argument can be one of
-\fIs\fRimple,
-\fIl\fRist,
-\fIo\fRutput,
-\fIt\fRhreads,
-\fIm\fRisc or
-\fIa\fRll.
-The argument can be shortened to one of the underlined letters as in: s|l|o|t|m|a.
+.IR s imple,
+.IR l ist,
+.IR o utput,
+.IR t hreads,
+.IR m "isc, or"
+.IR a ll.
+The argument can be shortened to one of the underlined letters as in:
+s\^|\^l\^|\^o\^|\^t\^|\^m\^|\^a.
 .TP
 .B \-\-info
 Print debugging info.
@@ -760,8 +764,6 @@ Print the procps-ng version.
 .B \-\-version
 Print the procps-ng version.
 .\" """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-.PD
-.PP
 .SH NOTES
 This
 .B ps
@@ -772,7 +774,6 @@ this
 .B ps
 any special permissions.
 .PP
-.PP
 CPU usage is currently expressed as the percentage of time spent running
 during the entire lifetime of a process.  This is not ideal, and\ it does not
 conform to the standards that
@@ -781,7 +782,7 @@ otherwise conforms to.  CPU usage is unlikely to add up to exactly 100%.
 .PP
 The SIZE and RSS fields don't count some parts of a process including the
 page tables, kernel stack, struct thread_info, and struct task_struct.  This
-is usually at least 20 KiB of memory that is always resident.  SIZE is the
+is usually at least 20\ KiB of memory that is always resident.  SIZE is the
 virtual size of the process (code+\:data+\:stack).
 .PP
 Processes marked <defunct> are dead processes (so\-called "zombies") that
@@ -791,7 +792,7 @@ will be destroyed by
 if the parent process exits.
 .PP
 If the length of the username is greater than the length of the display
-column, the username will be truncated. See the -o and -O formatting
+column, the username will be truncated.  See the \-o and \-O formatting
 options to customize length.
 .PP
 Commands options such as
@@ -802,13 +803,15 @@ display all processes with a TTY (generally the commands users are
 running) plus all processes owned by a user named "x".  If that user
 doesn't exist, then
 .B ps
-will assume you really meant "\fBps\fR \fIaux\fR".
+will assume you really meant
+.RB """" ps
+.IR aux """."
 .SH "PROCESS FLAGS"
 The sum of these values is displayed in the "F" column,
 which is provided by the
 .B flags
 output specifier:
-.IP
+.PP
 .RS 8
 .PD 0
 .TP 5
@@ -819,13 +822,12 @@ forked but didn't exec
 used super\-user privileges
 .PD
 .RE
-.PP
 .SH "PROCESS STATE CODES"
 Here are the different values that the
-.BR s , \ stat \ and \ state
+.BR s ", " stat " and " state
 output specifiers (header "STAT" or "S") will display to describe the state
 of a process:
-.IP
+.PP
 .RS 8
 .PD 0
 .TP 5
@@ -861,7 +863,7 @@ defunct ("zombie") process, terminated but not reaped by its parent
 For BSD formats and when the
 .B stat
 keyword is used, additional characters may be displayed:
-.IP
+.PP
 .RS 8
 .PD 0
 .TP 5
@@ -884,7 +886,6 @@ is multi-threaded (using CLONE_THREAD, like NPTL pthreads do)
 is in the foreground process group
 .PD
 .RE
-.PP
 .SH "OBSOLETE SORT KEYS"
 These keys are used by the BSD
 .B O
@@ -931,8 +932,6 @@ v	vsize	total VM size in KiB
 y	priority	kernel scheduling priority
 .\"K	stime	system time (conflict, system vs. start time)
 .TE
-.PP
-.PP
 .SH "AIX FORMAT DESCRIPTORS"
 This
 .B ps
@@ -942,7 +941,7 @@ formatting codes of
 and
 .IR printf (3).
 For example, the normal default output can be produced with this:
-\fBps \-eo "%p %y %x %c"\fR.
+.B ps \-eo """%p %y %x %c""\fR."
 The
 .B NORMAL
 codes are described in the next section.
@@ -967,7 +966,7 @@ l l l.
 .TE
 .SH "STANDARD FORMAT SPECIFIERS"
 Here are the different keywords that may be used to control the output
-format (e.g. with option
+format (e.g., with option
 .BR \-o )
 or to sort the selected processes with the GNU\-style
 .B \-\-sort
@@ -983,8 +982,8 @@ tries to recognize most of the keywords used in other implementations of
 .PP
 The following user\-defined format specifiers may contain
 spaces:
-.BR args , \ cmd , \ comm , \ command , \ fname , \ ucmd , \ ucomm ,
-.BR lstart , \ bsdstart , \ start .
+.BR args ", " cmd ", " comm ", " command ", " fname ", " ucmd ", " ucomm ,
+.BR lstart ", " bsdstart ", " start .
 .PP
 Some keywords may not be available for sorting.
 
@@ -997,8 +996,8 @@ Some keywords may not be available for sorting.
 .\"
 .TS
 expand;
-lB1 lB1 lBw(\n[ColSize]n)
-lB1 l1  l.
+l1B l1B lBw(\n[ColSize]n)
+l1B l1  l.
 CODE	HEADER	DESCRIPTION
 
 %cpu	%CPU	T{
@@ -1016,13 +1015,15 @@ machine, expressed as a percentage.  (alias
 T}
 
 args	COMMAND	T{
-command with all its arguments as a string. Modifications to the arguments
-may be shown.  The output in this column may contain spaces.  A process
-marked <defunct> is partly dead, waiting to be fully destroyed by its parent.
+command with all its arguments as a string.
+Modifications to the arguments may be shown.
+The output in this column may contain spaces.
+A process marked <defunct> is partly dead,
+waiting to be fully destroyed by its parent.
 Sometimes the process args will be unavailable; when this happens,
 .B ps
 will instead print the executable name in brackets.  (alias
-.BR cmd , \ command ).
+.BR cmd ", " command ).
 See also the
 .B comm
 format keyword, the
@@ -1053,14 +1054,14 @@ mask of the blocked signals, see
 .IR signal (7).
 According to the width of the field, a 32 or 64\-bit mask in hexadecimal
 format is displayed.  (alias
-.BR sig_block , \ sigmask ).
+.BR sig_block ", " sigmask ).
 T}
 
 bsdstart	START	T{
 time the command started.  If the process was started less than 24 hours ago,
 the output format is "\ HH:MM", else it is " Mmm:SS" (where Mmm is the three
 letters of the month).  See also
-.BR lstart , \ start , \ start_time ", and" \ stime .
+.BR lstart ", " start ", " start_time ", and " stime .
 T}
 
 bsdtime	TIME	T{
@@ -1070,8 +1071,9 @@ minutes of cpu time.
 T}
 
 c	C	T{
-processor utilization. Currently, this is the integer value of the percent
-usage over the lifetime of the process.  (see
+processor utilization.
+Currently, this is the integer value of the percent usage over the
+lifetime of the process.  (see
 .BR %cpu ).
 T}
 
@@ -1080,7 +1082,7 @@ mask of the caught signals, see
 .IR signal (7).
 According to the width of the field, a 32 or 64 bits mask in hexadecimal
 format is displayed.  (alias
-.BR sig_catch , \ sigcatch ).
+.BR sig_catch ", " sigcatch ).
 T}
 
 cgname	CGNAME	T{
@@ -1093,9 +1095,10 @@ T}
 
 class	CLS	T{
 scheduling class of the process.  (alias
-.BR policy , \ cls ).
+.BR policy ", " cls ).
 Field's possible values are:
-.IP "" 2
+.sp 1
+.in +9n
 \-	not reported
 .br
 TS	SCHED_OTHER
@@ -1113,13 +1116,15 @@ IDL	SCHED_IDLE
 DLN	SCHED_DEADLINE
 .br
 ?	unknown value
+.in
 T}
 
 cls	CLS	T{
 scheduling class of the process.  (alias
-.BR policy , \ cls ).
+.BR policy ", " cls ).
 Field's possible values are:
-.IP "" 2
+.sp 1
+.in +9n
 \-	not reported
 .br
 TS	SCHED_OTHER
@@ -1137,13 +1142,14 @@ IDL	SCHED_IDLE
 DLN	SCHED_DEADLINE
 .br
 ?	unknown value
+.in
 T}
 
 cmd	CMD	T{
 see
 .BR args .
 (alias
-.BR args , \ command ).
+.BR args ", " command ).
 T}
 
 comm	COMMAND	T{
@@ -1151,7 +1157,7 @@ command name (only the executable name).  Modifications to the command name
 will not be shown.  A process marked <defunct> is partly dead, waiting to be
 fully destroyed by its parent.  The output in this column may contain spaces.
 (alias
-.BR ucmd , \ ucomm ).
+.BR ucmd ", " ucomm ).
 See also the
 .B args format keyword,
 the
@@ -1179,7 +1185,7 @@ command	COMMAND	T{
 See
 .BR args .
 (alias
-.BR args , \ command ).
+.BR args ", " command ).
 T}
 
 cp	CP	T{
@@ -1240,14 +1246,14 @@ effective user name.  This will be the textual user ID, if it can be obtained
 and the field width permits, or a decimal representation otherwise.  The
 .B n
 option can be used to force the decimal representation.  (alias
-.BR uname , \  user ).
+.BR uname ", "  user ).
 T}
 
 f	F	T{
 flags associated with the process, see the
 .B PROCESS FLAGS
 section.  (alias
-.BR flag , \ flags ).
+.BR flag ", " flags ).
 T}
 
 fgid	FGID	T{
@@ -1266,14 +1272,14 @@ flag	F	T{
 see
 .BR f .
 (alias
-.BR f , \ flags ).
+.BR f ", " flags ).
 T}
 
 flags	F	T{
 see
 .BR f .
 (alias
-.BR f , \ flag ).
+.BR f ", " flag ).
 T}
 
 fname	COMMAND	T{
@@ -1310,11 +1316,12 @@ mask of the ignored signals, see
 .IR signal (7).
 According to the width of the field, a 32 or 64 bits mask in hexadecimal
 format is displayed.  (alias
-.BR sig_ignore , \ sigignore ).
+.BR sig_ignore ", " sigignore ).
 T}
 
 ipcns	IPCNS	T{
-Unique inode number describing the namespace the process belongs to. See namespaces(7).
+Unique inode number describing the namespace the process belongs to.
+See namespaces(7).
 T}
 
 label	LABEL	T{
@@ -1326,7 +1333,7 @@ T}
 
 lstart	STARTED	T{
 time the command started.  See also
-.BR bsdstart , \ start , \ start_time ", and" \ stime .
+.BR bsdstart ", " start ", " start_time ", and " stime .
 T}
 
 lsession	SESSION	T{
@@ -1340,7 +1347,7 @@ T}
 
 lwp	LWP	T{
 light weight process (thread) ID of the dispatchable entity (alias
-.BR spid , \ tid ).
+.BR spid ", " tid ).
 See
 .B tid
 for additional information.
@@ -1365,15 +1372,18 @@ The number of minor page faults that have occurred with this process.
 T}
 
 mntns	MNTNS	T{
-Unique inode number describing the namespace the process belongs to. See namespaces(7).
+Unique inode number describing the namespace the process belongs to.
+See namespaces(7).
 T}
 
 netns	NETNS	T{
-Unique inode number describing the namespace the process belongs to. See namespaces(7).
+Unique inode number describing the namespace the process belongs to.
+See namespaces(7).
 T}
 
 ni	NI	T{
-nice value. This ranges from 19 (nicest) to \-20 (not nice to others),
+nice value.
+This ranges from 19 (nicest) to \-20 (not nice to others),
 see
 .IR nice (1).
 (alias
@@ -1416,7 +1426,8 @@ see
 T}
 
 pending	PENDING	T{
-mask of the pending signals. See
+mask of the pending signals.
+See
 .IR signal (7).
 Signals pending on the process are distinct from signals pending on
 individual threads.  Use the
@@ -1447,7 +1458,8 @@ a number representing the process ID (alias
 T}
 
 pidns	PIDNS	T{
-Unique inode number describing the namespace the process belongs to. See namespaces(7).
+Unique inode number describing the namespace the process belongs to.
+See namespaces(7).
 T}
 
 pmem	%MEM	T{
@@ -1459,9 +1471,10 @@ T}
 
 policy	POL	T{
 scheduling class of the process.  (alias
-.BR class , \ cls ).
+.BR class ", " cls ).
 Possible values are:
-.IP "" 2
+.sp 1
+.in +9n
 \-	not reported
 .br
 TS	SCHED_OTHER
@@ -1479,6 +1492,7 @@ IDL	SCHED_IDLE
 DLN	SCHED_DEADLINE
 .br
 ?	unknown value
+.in
 T}
 
 ppid	PPID	T{
@@ -1504,22 +1518,22 @@ T}
 
 rss	RSS	T{
 resident set size, the non\-swapped physical memory that a task has used (in
-kiloBytes).  (alias
-.BR rssize , \ rsz ).
+kilobytes).  (alias
+.BR rssize ", " rsz ).
 T}
 
 rssize	RSS	T{
 see
 .BR rss .
 (alias
-.BR rss , \ rsz ).
+.BR rss ", " rsz ).
 T}
 
 rsz	RSZ	T{
 see
 .BR rss .
 (alias
-.BR rss , \ rssize ).
+.BR rss ", " rssize ).
 T}
 
 rtprio	RTPRIO	T{
@@ -1558,7 +1572,7 @@ T}
 
 sess	SESS	T{
 session ID or, equivalently, the process ID of the session leader.  (alias
-.BR session , \ sid ).
+.BR session ", " sid ).
 T}
 
 sgi_p	P	T{
@@ -1580,35 +1594,35 @@ sid	SID	T{
 see
 .BR sess .
 (alias
-.BR sess , \ session ).
+.BR sess ", " session ).
 T}
 
 sig	PENDING	T{
 see
 .BR pending .
 (alias
-.BR pending , \ sig_pend ).
+.BR pending ", " sig_pend ).
 T}
 
 sigcatch	CAUGHT	T{
 see
 .BR caught .
 (alias
-.BR caught , \ sig_catch ).
+.BR caught ", " sig_catch ).
 T}
 
 sigignore	IGNORED	T{
 see
 .BR ignored .
 (alias
-.BR ignored , \ sig_ignore ).
+.BR ignored ", " sig_ignore ).
 T}
 
 sigmask	BLOCKED	T{
 see
 .BR blocked .
 (alias
-.BR blocked , \ sig_block ).
+.BR blocked ", " sig_block ).
 T}
 
 size	SIZE	T{
@@ -1626,7 +1640,7 @@ spid	SPID	T{
 see
 .BR lwp .
 (alias
-.BR lwp , \ tid ).
+.BR lwp ", " tid ).
 T}
 
 stackp	STACKP	T{
@@ -1637,7 +1651,7 @@ start	STARTED	T{
 time the command started.  If the process was started less than 24 hours ago,
 the output format is "HH:MM:SS", else it is "\ \ Mmm\ dd" (where Mmm is a
 three\-letter month name).  See also
-.BR lstart , \ bsdstart , \ start_time ", and" \ stime .
+.BR lstart ", " bsdstart ", " start_time ", and " stime .
 T}
 
 start_time	START	T{
@@ -1646,7 +1660,7 @@ process was not started the same year
 .B ps
 was invoked, or "MmmDD" if it was not started the same day, or "HH:MM"
 otherwise.  See also
-.BR bsdstart , \ start , \ lstart ", and" \ stime .
+.BR bsdstart ", " start ", " lstart ", and " stime .
 T}
 
 stat	STAT	T{
@@ -1659,7 +1673,7 @@ T}
 
 state	S	T{
 see
-.BR s ". (alias" \ s ).
+.BR s ".\& (alias" \ s ).
 T}
 
 suid	SUID	T{
@@ -1720,7 +1734,7 @@ T}
 
 tid	TID	T{
 the unique number representing a dispatchable entity (alias
-.BR lwp , \ spid ).
+.BR lwp ", " spid ).
 This value may also appear as: a process ID (pid); a process group ID (pgrp);
 a session ID for the session leader (sid); a thread group ID for the thread
 group leader (tgid); and a tty process group ID for the process group leader
@@ -1739,7 +1753,7 @@ T}
 
 tname	TTY	T{
 controlling tty (terminal).  (alias
-.BR tt , \ tty ).
+.BR tt ", " tty ).
 T}
 
 tpgid	TPGID	T{
@@ -1748,31 +1762,32 @@ connected to, or \-1 if the process is not connected to a tty.
 T}
 
 trs	TRS	T{
-text resident set size, the amount of physical memory devoted to executable code.
+text resident set size,
+the amount of physical memory devoted to executable code.
 T}
 
 tt	TT	T{
 controlling tty (terminal).  (alias
-.BR tname , \ tty ).
+.BR tname ", " tty ).
 T}
 
 tty	TT	T{
 controlling tty (terminal).  (alias
-.BR tname , \ tt ).
+.BR tname ", " tt ).
 T}
 
 ucmd	CMD	T{
 see
 .BR comm .
 (alias
-.BR comm , \ ucomm ).
+.BR comm ", " ucomm ).
 T}
 
 ucomm	COMMAND	T{
 see
 .BR comm .
 (alias
-.BR comm , \ ucmd ).
+.BR comm ", " ucmd ).
 T}
 
 uid	UID	T{
@@ -1786,7 +1801,7 @@ uname	USER	T{
 see
 .BR euser .
 (alias
-.BR euser , \ user ).
+.BR euser ", " user ).
 T}
 
 unit	UNIT	T{
@@ -1798,15 +1813,17 @@ user	USER	T{
 see
 .BR euser .
 (alias
-.BR euser , \ uname ).
+.BR euser ", " uname ).
 T}
 
 userns	USERNS	T{
-Unique inode number describing the namespace the process belongs to. See namespaces(7).
+Unique inode number describing the namespace the process belongs to.
+See namespaces(7).
 T}
 
 utsns	UTSNS	T{
-Unique inode number describing the namespace the process belongs to. See namespaces(7).
+Unique inode number describing the namespace the process belongs to.
+See namespaces(7).
 T}
 
 uunit	UUNIT	T{
@@ -1836,8 +1853,6 @@ T}
 
 .TE
 .\" #######################################################################
-.PP
-.PP
 .SH "ENVIRONMENT VARIABLES"
 The following environment variables could affect
 .BR ps :
@@ -1849,12 +1864,12 @@ Override default display width.
 Override default display height.
 .TP
 .B PS_PERSONALITY
-Set to one of posix, old, linux, bsd, sun, digital...  (see section
+Set to one of posix, old, linux, bsd, sun, digital...\& (see section
 .B PERSONALITY
 below).
 .TP
 .B CMD_ENV
-Set to one of posix, old, linux, bsd, sun, digital...  (see section
+Set to one of posix, old, linux, bsd, sun, digital...\& (see section
 .B PERSONALITY
 below).
 .TP
@@ -1868,8 +1883,8 @@ Date format.
 Not currently supported.
 .TP
 .B PS_FORMAT
-Default output format override. You may set this to a format
-string of the type used for the
+Default output format override.
+You may set this to a format string of the type used for the
 .B \-o
 option.
 The
@@ -1889,7 +1904,10 @@ When set to "on", acts as
 Don't find excuses to ignore bad "features".
 .TP
 .B _XPG
-Cancel \fBCMD_ENV\fR=\fIirix\fR non\-standard behavior.
+Cancel
+.BR CMD_ENV =\c
+.I irix
+non\-standard behavior.
 .PP
 In general, it is a bad idea to set these variables.  The one exception is
 .B CMD_ENV
@@ -1898,7 +1916,6 @@ or
 which could be set to Linux for normal systems.  Without that setting,
 .B ps
 follows the useless and bad parts of the Unix98 standard.
-.PP
 .SH "PERSONALITY"
 .TS
 l	l.
@@ -1928,15 +1945,11 @@ unix	standard
 unix95	standard
 unix98	standard
 .TE
-.PP
-.PP
 .SH "SEE ALSO"
 .BR pgrep (1),
 .BR pstree (1),
 .BR top (1),
 .BR proc (5).
-.PP
-.PP
 .SH STANDARDS
 This
 .B ps
@@ -1954,36 +1967,35 @@ X/Open System Interfaces Extension [UP\ XSI]
 .IP 5 4
 ISO/IEC 9945:2003
 .PD
-.PP
 .SH AUTHOR
 .B ps
 was originally written by
-.UR lankeste@\:fwi.\:uva.\:nl
+.MT lankeste@\:fwi.\:uva.\:nl
 Branko Lankester
-.UE .
-.UR johnsonm@\:redhat.\:com
-Michael K. Johnson
-.UE
+.ME .
+.MT johnsonm@\:redhat.\:com
+Michael K.\& Johnson
+.ME
 re\-wrote it significantly to use the proc filesystem, changing a few things
 in the process.
-.UR mjshield@\:nyx.\:cs.\:du.\:edu
+.MT mjshield@\:nyx.\:cs.\:du.\:edu
 Michael Shields
-.UE
+.ME
 added the pid\-list feature.
-.UR cblake@\:bbn.\:com
+.MT cblake@\:bbn.\:com
 Charles Blake
-.UE
+.ME
 added multi\-level sorting, the dirent\-style library, the device
 name\-to\-number mmaped database, the approximate binary search directly on
 System.map, and many code and documentation cleanups.  David Mossberger\-Tang
 wrote the generic BFD support for psupdate.
-.UR albert@\:users.\:sf.\:net
+.MT albert@\:users.\:sf.\:net
 Albert Cahalan
-.UE
+.ME
 rewrote ps for full Unix98 and BSD support, along with some ugly hacks for
 obsolete and foreign syntax.
 .PP
 Please send bug reports to
-.UR procps@\:freelists.\:org
-.UE .
+.MT procps@\:freelists.\:org
+.ME .
 No subscription is required or suggested.
diff --git a/skill.1 b/skill.1
index 8ef7683..29a5667 100644
--- a/skill.1
+++ b/skill.1
@@ -1,19 +1,19 @@
-'\" t
-.\" (The preceding line is a note to broken versions of man to tell
-.\" them to pre-process this man page with tbl)
-.\" Man page for skill and snice.
-.\" Licensed under version 2 of the GNU General Public License.
-.\" Written by Albert Cahalan, converted to a man page by
-.\" Michael K. Johnson
-.\"
-.TH SKILL 1 "October 2011" "procps-ng" "User Commands"
+.ig
+Written by Albert Cahalan, converted to a man page by Michael K. Johnson
+
+This manpage is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+..
+.TH SKILL 1 "2018-05-31" "procps-ng" "User Commands"
 .SH NAME
 skill, snice \- send a signal or report process status
 .SH SYNOPSIS
 .B skill
 .RI [ signal ]
 .RI [ options ]
-.I expression
+.I expression
 .br
 .B snice
 .RI [ "new priority" ]
@@ -81,7 +81,7 @@ The next expression is a command name.
 \fB\-\-ns \fIpid\fR
 Match the processes that belong to the same namespace as pid.
 .TP
-\fB\-\-nslist \fIns,...\fR
+\fB\-\-nslist \fIns\/\fR,\,\fI...\/\fR
 list which namespaces will be considered for the --ns option.
 Available namespaces: ipc, mnt, net, pid, user, uts.
 .PD
@@ -91,7 +91,7 @@ The behavior of signals is explained in
 manual page.
 .SH EXAMPLES
 .TP
-.B snice -c seti -c crack +7
+.B snice \-c seti \-c crack +7
 Slow down seti and crack commands.
 .TP
 .B skill \-KILL \-t /dev/pts/*
@@ -110,12 +110,12 @@ Stop three users.
 .SH STANDARDS
 No standards apply.
 .SH AUTHOR
-.UR albert@users.sf.net
+.MT albert@users.sf.net
 Albert Cahalan
-.UE
+.ME
 wrote skill and snice in 1999 as a replacement for a non-free
 version.
 .SH "REPORTING BUGS"
 Please send bug reports to
-.UR procps@freelists.org
-.UE
+.MT procps@freelists.org
+.ME
-- 
2.6.4.windows.1

